pkg/init/cmd/service/main.go

```diff
@@ -66,8 +70,17 @@ func main() {
 
 	args := flag.Args()
 	if len(args) < 1 {
-		systemInitCmd(args)
-		os.Exit(0)
+		// check if called form startup scripts
+		command := os.Args[0]
+		switch {
+		case strings.Contains(command, "onboot"):
+			os.Exit(runcInit(onbootPath))
+		case strings.Contains(command, "onshutdown"):
+			os.Exit(runcInit(shutdownPath))
+		case strings.Contains(command, "containerd"):
```

Comment:
This is how they were previously in `onshutdown`; they could be changed in a followup PR.